cli/src/format.rs

```diff
@@ -95,6 +95,65 @@ impl Format for PolicyEvaluationResponse {
     }
 }
 
+impl Format for PolicyEvaluationResponseRaw {
+    fn pretty<W: Write>(&self, writer: &mut W) {
+        let _ = writeln!(writer);
+
+        // Print summary status.
+        let status = if self.is_failure {
+            style("FAILURE").red()
+        } else if self.incomplete_packages_count > 0 {
+            style("INCOMPLETE").yellow()
+        } else {
+            style("SUCCESS").green()
+        };
+        let _ = writeln!(writer, "Phylum Supply Chain Risk Analysis — {status}\n");
+
+        // Print number of unprocessed packages.
+        if self.incomplete_packages_count > 0 {
+            let pluralization = if self.incomplete_packages_count == 1 { "" } else { "s" };
+            let unprocessed_text =
+                format!("{} unprocessed package{}", self.incomplete_packages_count, pluralization);
+            let incomplete_message = format!(
+                "The analysis contains {}, preventing a complete risk analysis. Phylum is \
+                 currently processing these packages and should complete soon. Please wait for up \
+                 to 30 minutes, then re-run the analysis.\n",
+                style(unprocessed_text).yellow(),
+            );
+            let _ = writeln!(writer, "{}", textwrap::fill(&incomplete_message, 80));
+        }
+
+        // Write summary for each issue.
+        for package in &self.dependencies {
+            for rejection in package.rejections.iter().filter(|rejection| !rejection.suppressed) {
+                let severity = match rejection.source.domain.as_str() {
+                    "author" => "AUT",
+                    "engineering" => "ENG",
+                    "malicious_code" | "malicious" => "MAL",
+                    "vulnerability" => "VUL",
+                    "license" => "LIC",
+                    _ => "UNK",
+                };
+                let message = format!("[{severity}] {}", rejection.title);
+
+                let colored = match rejection.source.severity.as_str() {
+                    "low" | "info" => style(message).green(),
```

Comment:
Severity is weird...

If the policy rejects a `low` or `info` issue, the result is a failed job. Printing that in green seems counterintuitive.

In an ideal world, I think the policy should tell us the severity. But we're not there yet... It might be better to simply print every rejection in red since that's what we've got right now.

Reply:
The UI also shows things as green even when they're technically an "issue".

I think this is just consistent with the rest of our platform.

Reply:
We're working on changing this in the UI atm. Basically a transition to triaging at the package level, instead of issue level. So I'd say we should do whatever is right here, and let the rest of the platform catch up.

Reply:
After discussing with @louislang, I'm going to resolve this conversation and we can leave this as is.

There is still quite a bit of uncertainty about how this will look when the UI is updated. So matching the UI today and tweaking this code if/when the UI changes is a good strategy moving forward.